src/csm_bot/main.py

```diff
@@ -71,6 +71,16 @@ async def handle_event_log(self, event: Event, context: ContextTypes.DEFAULT_TYP
                            .union(context.bot_data.get("channel_ids", set())))
         if "nodeOperatorId" in event.args:
             chats = context.bot_data["no_ids_to_chats"].get(str(event.args["nodeOperatorId"]), set())
+        elif event.event == "DistributionDataUpdated":
```

Comment:
pls suggest an architecture change to not make such ifs for every event needed similar logic